cli/tests/unit/read_text_file_test.ts

```diff
@@ -164,7 +164,13 @@ Deno.test(
     const bytes = new Uint8Array(kStringMaxLengthPlusOne);
     const filePath = "cli/tests/testdata/too_big_a_file.txt";
 
-    Deno.writeFileSync(filePath, bytes);
+    try {
+      Deno.writeFileSync(filePath, bytes);
+    } catch (e) {
```

Comment:
Do we have a way to check the specific error instead of returning on all errors?

Reply:
It's OS specific so I'm not sure what will the error be on other OSes.